### ISSUE TITLE
Add support to local comflowyspace GUI connect to remote server comfyUI

### ISSUE THREAD

**Issue by VinciShark** [6174/comflowyspace#222]
I would be wonderful if I can use comflowyspace on my macbook air but connect to Tesla 100 GPU on my company's server or 4090Ti desktop in my office.

**Comment by 6174** [comment]
Yes, this is 100% sure in our plan.  But at first we need make the app go through alpha + beta stages .

**Comment by mariorodriguespt** [comment]
Also need this
